app/static/css/style.css

```diff
@@ -19,6 +20,7 @@
 
 * {
   box-sizing: border-box;
+  font-family: 'Inter';
```

Comment:
This and all the below fonts don't make sense. We're downloading all three fonts but overriding everything to use one.